modules/core/src/main/scala/org/tessellation/domain/trust/storage/OrdinalTrustMap.scala

```diff
@@ -0,0 +1,26 @@
+package org.tessellation.domain.trust.storage
+
+import org.tessellation.domain.trust.storage.configuration.snapshotInterval
+import org.tessellation.schema.SnapshotOrdinal
+
+case class OrdinalTrustMap(
+  ordinal: SnapshotOrdinal,
+  peerLabels: PublicTrustMap,
+  trust: TrustMap
+)
+
+object OrdinalTrustMap {
+
+  implicit val ordering: Ordering[OrdinalTrustMap] = Ordering.by(_.ordinal)
```

Comment:
I'd create an `Order` from cats instead of scala `ordering`

Reply:
Changed to `implicit val order: Order[OrdinalTrustMap] = Order[SnapshotOrdinal].contramap(_.ordinal)`